src/cli.rs

```diff
@@ -13,18 +13,24 @@ pub struct Args {
     pub target: Option<Target>,
     pub target_dir: Option<PathBuf>,
     pub docker_in_docker: bool,
+    pub project_dir: Option<String>,
 }
 
 pub fn parse(target_list: &TargetList) -> Args {
     let mut channel = None;
     let mut target = None;
+    let mut project_dir: Option<String> = None;
     let mut target_dir = None;
     let mut sc = None;
     let mut all: Vec<String> = Vec::new();
 
     {
         let mut args = env::args().skip(1);
         while let Some(arg) = args.next() {
+            if all.last().unwrap_or(&"".to_string()) == "--manifest-path" {
```

Comment:
Why `all.last()` when we are using an iterator? Use the same pattern as for the other flags below.

Reply:
fix